common/djangoapps/edraak_i18n/middleware.py

```diff
@@ -30,6 +31,9 @@ def process_request(self, request):
         if not settings.FEATURES.get('EDRAAK_I18N_LOCALE_MIDDLEWARE'):
             return
 
+        if 'HTTP_X_API_ACCEPT_LANGUAGE' in request.META:
+            request.session[LANGUAGE_SESSION_KEY] = request.META['HTTP_X_API_ACCEPT_LANGUAGE']
```

Comment:
We should not be touching the session here. So please use the `META` value instead.

@Salomari1987 @@Fahminajjar @devalih correct?

```suggestion
            # Override the API accept language
            request.META[HTTP_ACCEPT_LANGUAGE] = request.META['HTTP_X_API_ACCEPT_LANGUAGE']
            return
```

Reply:
Done